src/main/java/Duke.java

```diff
@@ -1,10 +1,107 @@
+import java.util.Scanner;
+import java.util.ArrayList; 
+
 public class Duke {
-    public static void main(String[] args) {
+    void logo(){
         String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
     }
+    static void greeting(){
+        String output = "    ____________________________________________________________\n" +
+        "     Hello! I'm Duke\n" + 
+        "     What can I do for you?\n" +
+        "    ____________________________________________________________\n";
+        System.out.println(output);
+    }
+    static String addDoubleLine(String str){
```

Comment:
Seems like quite a tedious process, especially when you increase the OOP later on, you can try to just print out a line before and after parsing the string